packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -55,6 +55,7 @@ storiesOf('Components|Brand', module)
           minWidth={minWidthInput}
           maxWidth={maxWidthInput}
           svg={svgs[svgChoice]}
+          borderBottom
```

Comment:
Can this be controlled using knobs? I'd like to be able to set `borderTop` and `borderBottom`